### DISCUSSION TITLE
Access enable/disable_radial in EntityMethodInterface

### DISCUSSION THREAD

**Discussion by Cervicia** [MattFiler/LegendPlugin#154] [Q&A] [answered]
Hello,
I was wondering if it is possible to access the _EntityMethodInterface_ class? My goal was to control the radial menu using the _disable_radial_ and _enable_radial_ functions. Unfortunately I haven't worked with Unreal Engine yet and therefore I don't know much about entities. The wiki says that the _EntityMethodInterface_ class is the base class for all entities, but I didn't find such a function in the GUI of the Commands Editor. What exactly does this class mean and how would it be possible to control the access to the radial menu? Unfortunately I also couldn't find the script that is responsible for controlling the radial menu, is there a good way to find it?

**Comment by MattFiler** [comment, marked as answer]
Interfaces are what entities base from, with `EntityMethodInterface` being the base interface for all entities. You can see all interface parameters [here](https://myfiles.mattfiler.co.uk/all_nodes_with_params3.html) - although be aware that not every entity implements them. I'm still working on displaying this better in the editor UI - for now, it only shows parameters that each individual entity implements itself, which could do with some improvement.

To enable the radial menu, you can add a `ToggleFunctionality` entity, which has options for disabling the radial menu, and specific components of it.
